search/search_repository_whitebox_test.go

```diff
@@ -240,55 +229,51 @@ func stringInSlice(str string, list []string) bool {
 }
 
 func (s *searchRepositoryWhiteboxTest) TestSearchByID() {
+	req := &http.Request{Host: "localhost"}
+	params := url.Values{}
+	ctx := goa.NewContext(context.Background(), nil, req, params)
 
-	models.Transactional(s.DB, func(tx *gorm.DB) error {
-		req := &http.Request{Host: "localhost"}
-		params := url.Values{}
-		ctx := goa.NewContext(context.Background(), nil, req, params)
-
-		wir := workitem.NewWorkItemRepository(tx)
+	wir := workitem.NewWorkItemRepository(s.DB)
 
-		workItem := workitem.WorkItem{Fields: make(map[string]interface{})}
+	workItem := workitem.WorkItem{Fields: make(map[string]interface{})}
 
-		workItem.Fields = map[string]interface{}{
-			workitem.SystemTitle:       "Search Test Sbose",
-			workitem.SystemDescription: rendering.NewMarkupContentFromLegacy("Description"),
-			workitem.SystemCreator:     "sbose78",
-			workitem.SystemAssignees:   []string{"pranav"},
-			workitem.SystemState:       "closed",
-		}
+	workItem.Fields = map[string]interface{}{
+		workitem.SystemTitle:       "Search Test Sbose",
+		workitem.SystemDescription: rendering.NewMarkupContentFromLegacy("Description"),
+		workitem.SystemCreator:     "sbose78",
+		workitem.SystemAssignees:   []string{"pranav"},
+		workitem.SystemState:       "closed",
+	}
 
-		createdWorkItem, err := wir.Create(ctx, space.SystemSpace, workitem.SystemBug, workItem.Fields, s.modifierID)
-		if err != nil {
-			s.T().Fatalf("Couldn't create test data: %+v", err)
-		}
+	createdWorkItem, err := wir.Create(ctx, space.SystemSpace, workitem.SystemBug, workItem.Fields, s.modifierID)
```

Comment:
May I ask why you don't use a test fixture here?

Reply:
I can do that, indeed

Reply:
done, too.